index.html

```diff
@@ -64,113 +78,138 @@ <h2 class="title-about">I'm a Software Developer</h2>
-                    <li>JavaScript</li>                
-                </ul>
-                <a href="#" class="btn-works">See Project</a>
-            </div>
-            <div class="card">
-                <h1 class="card-title">Profesional Art Printing Data</h1>
-                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
-                <ul class="card-lang">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>JavaScript</li>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                    has been the industry's standard dummy text ever since the 1500s, when an unknown printer took 
+                    a standard dummy text.
+                </p>
+
+                <ul class="tech">
+                    <li><a href="#">CSS</a></li>
+                    <li><a href="#">HTML</a></li>
+                    <li><a href="#">Bootstrap</a></li>
+                    <li><a href="#">Ruby</a></li>
                 </ul>
-                <a href="#" class="btn-works">See Project</a>
+
+                <a href="#" class="btn-works-1">See Projects</a>
             </div>
         </div>
 
+        
     </section>
 
+<!-- Project Cards -->
+    <section class="card-container">
+        
+        <div class="card card-1">
+            <h1 class="card-title">Profesional Art Printing Data</h1>
+            <article class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</article>
+            <ul class="card-lang">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Ruby</li>
+            </ul>
+            <a href="#" class="btn-works">See Project</a>
+        </div>
+        <div class="card card-2">
+            <h1 class="card-title">Profesional Art Printing Data</h1>
+            <h2 class="card-desk-title">Data Dashboard Healthcare</h2>
+            <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="card-lang">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <a href="#" class="btn-works">See Project</a>
+        </div>
+        <div class="card card-3">
+            <h1 class="card-title">Profesional Art Printing Data</h1>
+            <h2>Web Portfolio</h2>
+            <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="card-lang">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
```

Comment:
Kindly implement the following changes for the project section of your portfolio to hep you get a better UI :

- [ ] Please adjust the `hr` to be close to the title as it is in the Figma template given
- [ ] kindly adjust the size of the image to be sizable as close to the figma template as given
- [ ] Also kindly use the font color given as it is in the Figma template
- [ ] Please use the right width for all the project cards similar to what we have in the design

Figma design :
![image](https://github.com/ulises2607/Portfolio/assets/20537712/3c6f2bf2-aa7c-4688-9228-a57219ad01a4)

Your design :
![image](https://github.com/ulises2607/Portfolio/assets/20537712/9ca0294c-5738-4701-9134-896cb3c782de)

![image](https://github.com/ulises2607/Portfolio/assets/20537712/866d1c03-977a-43a0-97fd-142da61cc837)